setup.py

```diff
@@ -18,9 +18,8 @@
 
 install_requirements = [
     'click >= 7.0',
-    # Temporary to suppress paramiko Blowfish warning which breaks CI.
-    # Pinning cryptography should not be needed after paramiko 2.11.0.
-    'cryptography == 36.0.2',
+    # Pinning cryptography is not needed after paramiko 2.11.0. Correct it
+    'cryptography >= 1.0.0',
```

Comment:
If pinning is not needed, I think we can just delete this line, because we didn't depend on cryptography directly from code.

Reply:
apologies for the delay 🥺

can you resolve the conflict as well? thanks

Reply:
> apologies for the delay 🥺
> 
> can you resolve the conflict as well? thanks

Do you mean the "Resolve conversation" button below?

Reply:
No, I mean, this PR can not be merged now, cause it has conflicts.

<img width="460" alt="image" src="https://github.com/dbcli/mycli/assets/9675939/75111aec-0a67-45f8-a27d-bedc74399c0d">

Can you rebase from main branch, or resubmit one? thanks.

Reply:
> No, I mean, this PR can not be merged now, cause it has conflicts.
> <img alt="image" width="460" src="https://private-user-images.githubusercontent.com/9675939/308893233-75111aec-0a67-45f8-a27d-bedc74399c0d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzNjU4NjIsIm5iZiI6MTcwOTM2NTU2MiwicGF0aCI6Ii85Njc1OTM5LzMwODg5MzIzMy03NTExMWFlYy0wYTY3LTQ1ZjgtYTI3ZC1iZWRjNzQzOTljMGQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDJUMDc0NjAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2VhZTcyYjIxYjBiYzZkZWEyZWJjOGRkZjlmYTM5MmEwMmQ2ZmZjYWFjMWE4MzUzZTNmYjAzN2FlYjIyYmNkNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.xAgIMGsX4_HCS7qPxUwo5PTOS5EoMFcZQEDcOM5F3LQ">
> 
> Can you rebase from main branch, or resubmit one? thanks.

Oh，please allow me some time.